Packs/Zafran/Integrations/Zafran/Zafran.yml

```
@@ -0,0 +1,136 @@
+commonfields:
+  id: Zafran API
+  version: -1
+name: Zafran API
+display: Zafran API
+category: Utilities
+description: 'The Zafran API provides a programmatic way to interact with the Zafran Risk & Mitigation Platform for various use cases.  Using the API requires an   API token  , which can be obtained by contacting the Zafran support team at support@zafran.io.  API authorization is based on a bearer token that should be added as a header to all requests made for the AP, in the following way:  `Authorization: Bearer  token ` '
```

```suggestion
description: 'The Zafran API provides a programmatic way to interact with the Zafran Risk & Mitigation Platform for various use cases.  Using the API requires an API token '
```
`wich can be obtained by contacting the Zafran support team at support@zafran.io.` - add to the `Zafran_description.md` file instructions on how to obtain API Key and configure the integration.


`API authorization is based on a bearer token that should be added as a header to all requests made for the AP, in the following way:  Authorization: Bearer  token ` - I dont think this  revevant to the customers